core/templates/dev/head/components/response_header_directive.html

```diff
@@ -62,15 +62,18 @@
 
   <br class="break-in-mobile">
   <span ng-if="getOutcome()">→</span>
-  <span ng-if="getOutcome() && !isOutcomeLooping() && !isCreatingNewState()"
+  <span ng-if="isCorrect() && getInQuestionMode()">
+    Continue
+  </span>
+  <span ng-if="getOutcome() && !isOutcomeLooping() && !isCreatingNewState() && !getInQuestionMode()"
         ng-click="navigateToState(getOutcome().dest)" class="oppia-nested-link">
     <[getOutcome().dest]>
   </span>
   <span ng-if="getOutcome() && !isOutcomeLooping() && isCreatingNewState()">
     <em ng-if="getOutcome().newStateName">(<[getOutcome().newStateName]>)</em>
     <em ng-if="!getOutcome().newStateName">Nowhere yet...</em>
   </span>
-  <span ng-if="getOutcome() && isOutcomeLooping()">
+  <span ng-if="(getOutcome() && isOutcomeLooping()) || (!isCorrect() && !getOutcome().dest && getInQuestionMode())">
```

Comment:
Do we have checks (both backend and frontend) to ensure that an answer group can't both be correct and looping?

Reply:
There aren't, but in the case of a correct answer, the destination wouldn't be checked in the player. Correct answer would always lead to the next card, so is a check like that necessary?

Reply:
Yes I think we should have a check, if this case is unexpected and should not occur in practice. Helps raise any issues early and give us better control of what is being stored in the backend models. Can we add one?

Reply:
Done, hid the  outcome destination editor in question editor.